odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -297,8 +298,7 @@ def pdhg(x, f, g, L, tau, sigma, niter, **kwargs):
         # Over-relaxation in the primal variable x
         x_relax.lincomb(1 + theta, x, -theta, x_old)
 
-        if callback is not None:
-            callback(x)
+        call_callback(x, dual_iterate=y)
```

Comment:
would this not be `call_callback(callback, x, dual_iterate=y)`?

Reply:
You are totally correct!